src/main/java/pl/sdacademy/carrental/configuration/DomainValues.java

```diff
@@ -0,0 +1,11 @@
+package pl.sdacademy.carrental.configuration;
+
+import org.springframework.stereotype.Component;
+
+@Component
+// TODO: 18/06/2020 it would probably be better to store these values in DB
+public class DomainValues {
```

Comment:
if you want to use class for consts -> make it final. Other approach is to define those in an interface

Reply:
good point! done